lib/services/ios-log-filter.ts

```diff
@@ -16,7 +16,7 @@ export class IOSLogFilter extends iOSLogFilterBase.IOSLogFilter implements Mobil
 
 	public filterData(data: string, logLevel: string, pid?: string): string {
 		data = super.filterData(data, logLevel, pid);
-		if (pid && data && data.indexOf(`[${pid}]`) === -1) {
+		if (pid && data && data.search(new RegExp(`[\\[\\s]${pid}[\\]\\s]`)) === -1) {
```

Comment:
IMO either choose RegExp or indexOf for consistency [in the code](https://github.com/telerik/mobile-cli-lib/pull/1016/files#diff-6970fd71154619adb4a5ac4fc61de7fcR11)

Reply:
I used regex to avoid nested or complex "if" statements:
```javascript
if(data && pid){
  if(data.indexOf(` ${pid} `) ||  data.indexOf(`[${pid}]`)) {
  }
}
```
or
```javascript
if(data && pid && (data.indexOf(` ${pid} `) ||  data.indexOf(`[${pid}]`)) {
}
```

Reply:
The statement `` (data.indexOf(` ${pid} `) ||  data.indexOf(`[${pid}]`) `` can be extracted into a separate method and called everywhere needed.
We'd avoid complexity and gain readability IMO